NGINX Writing Day Plan (#2324)
writethedocs/www: docs/conf/portland/2025/writing-day.md

- **Ask questions!** If you encounter challenges with new concepts and tools, you are in a room surrounded by friendly people from diverse backgrounds and experiences. You can always ask the Welcome Wagon or Registration staff and volunteers. We will help you connect with the right person!
- **View the project list.** Check out the projects that have been submitted in advance. Keep in mind that some attendees choose to announce their projects during Writing Day itself, so these projects might not appear on the list in advance.

**We're excited to have another wonderful Writing Day!**

## Project List

### Enhance Your Portfolio! Contribute to an Open Source Repository

Project organizer: Mike Jang, he/him.

While we've open sourced our enterprise [documentation repository](https://github.com/nginx/documentation), we've done something different. We've set up [Good first issues](https://github.com/nginx/documentation/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22) that do not require Git knowledge.

We love all types of contributions. Examples:

> - We have not copmleted our move towards "sentence case" in our headings

Reviewer: ```suggestion
- We have not completed our move towards "sentence case" in our headings
```

- We are missing alternative text for a few of our screenshots

> Of course, if you know Git, we also would love your contributions. 

Reviewer 2: This is kinda dangling? Unless I'm missing something?

Author: My intent: motivate writers who don't know Git -- but continue to welcome contributions from Git experts

Reviewer 2: Maybe something like "Whether you've never used git or you're already an expert, contributions welcome" but don't let this be a blocker to anything.

Author: ```suggestion
If you've never used git or if you're already an expert, we welcome your contributions. 
```

> If you're a technical guru, our Good first issues give you the chance to test our how-tos and tutorials! If you remember Manny Silva's [Docs as Tests](https://www.docsastests.com/), several of our Good first issues include testable procedures.

Reviewer 2: Might be worth rethinking guru as well?

Author: I'm open to suggestions

Reviewer 2: s/guru/expert/ ?

But also not a blocker

Author: ```suggestion
If you're a technical expert, our Good first issues give you the chance to test our how-tos and tutorials! If you remember Manny Silva's [Docs as Tests](https://www.docsastests.com/), several of our Good first issues include testable procedures.
```
